renovate.json

```diff
@@ -26,6 +26,7 @@
     "ghcr.io/keptn/lifecycle-operator",
     "ghcr.io/keptn/scheduler",
     "ghcr.io/keptn/functions-runtime",
```

Comment:
I guess it can be removed

Reply:
i removed everything except metrics-operator to be able to get updates through renovate for everything now